pkg/couchdb/couchdb.go

```diff
@@ -214,6 +214,9 @@ func (j JSONDoc) Valid(field, value string) bool {
 		}
 
 		values := strings.Split(value, "/")
+		if len(values) != 2 {
```

Comment:
This doesn't directly concern the PR, it is to avoid errors due to a wrong formatting of the referenced_by values. I wanted to make a separate PR but I forgot…